### PR TITLE
log alias to app logs command and shorthand to follow and lines

### PR DESCRIPTION
**Pull request by diegofernandes**
This change is [Reviewable](https://reviewable.io/reviews/luizalabs/teresa/380)

### CODE REVIEW

File: pkg/client/cmd/app.go [hunk 1]
```diff
@@ -617,8 +618,8 @@ func init() {
 	appEnvUnSetCmd.Flags().String("app", "", "app name")
 	appEnvUnSetCmd.Flags().Bool("no-input", false, "unset env vars without warning")
 	// App logs
-	appLogsCmd.Flags().Int64("lines", 10, "number of lines")
-	appLogsCmd.Flags().Bool("follow", false, "follow logs")
+	appLogsCmd.Flags().Int64P("lines", "l", 10, "number of lines")
```
Comment (yagonobre) [inline pkg/client/cmd/app.go]: maybe `n` instead of `l`?

**Comment by drgarcia1986** [comment]
Hi, @diegofernandes thank you!
Could you add a Changelog update about this?

File: pkg/client/cmd/app.go (hunk 1 above)
Comment (diegofernandes) [reply pkg/client/cmd/app.go]: Yeap sounds better.

**Comment by diegofernandes** [comment]
Hi @drgarcia1986, sure!

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,9 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ### Changed
 - Back to Godep for dependencies management
 - Update `client-go` lib to version 4.0
+### Added
+- alias `log` to `app logs`
```
Comment (yagonobre) [inline CHANGELOG.md]: I think that the alias is `app log`

Comment (diegofernandes) [reply CHANGELOG.md]: yeah, true. 😬

**Pull request merged**